cypress/integration/field/reviewChanges.test.js

```diff
@@ -0,0 +1,128 @@
+import {uuid} from '@sanity/uuid'
+import client from '../../helpers/sanityClientSetUp'
+
+const DOC_ID = `drafts.${uuid()}`
+
+const doc = {
+  _id: DOC_ID,
+  _type: 'reviewChanges',
+  stringWithDefaultDiffComponent: undefined,
+  stringWithCustomDiffComponent: undefined,
+  stringWithRenderDiffComponent: undefined,
+  numberWithDefaultDiffComponent: undefined,
+  numberWithCustomDiffComponent: undefined,
+  numberWithRenderDiffComponent: undefined,
+  booleanWithDefaultDiffComponent: undefined,
+  booleanWithCustomDiffComponent: undefined,
+  booleanWithRenderDiffComponent: undefined,
+  slugWithDefaultDiffComponent: undefined,
+  slugWithCustomDiffComponent: undefined,
+  slugWithRenderDiffComponent: undefined,
+}
```

Comment:
We could type this as `SanityDocument`

```suggestion
import {SanityDocument} from '@sanity/types'

const doc: SanityDocument = {
  _id: DOC_ID,
  _type: 'reviewChanges'
```